src/plus/gk/checkin.ts

```diff
@@ -176,6 +177,7 @@ export function getSubscriptionFromCheckIn(
 			new Date(license.latestEndDate),
 			license.latestStatus === 'cancelled',
 			license.nextOptInDate ?? data.nextOptInDate,
+			license.latestStatus === 'trial' || license.latestStatus === 'in_trial',
```

Comment:
This is *probably* fine, but I'm not 100% sure if we can rely on `latestStatus` coming through as 'trial' for trial licenses the way we are hoping for it to here. Since effective licenses from the api check-in response were always intended to be for trials, and paid licenses were always intended to be non-trial, we could probably just get away with using `true` in effective licenses and `false` for paid ones when setting them up from the check-in response.

Reply:
hmmm, okay, I just uploaded a commit with this